### PR TITLE
F#12 fire rescue box

### LINKED ISSUE #12: Upload models of fire-resque box
Upload blender models of fire-resque box into robert_gazebo/cad/ directory

Upload the visual models of fire-resque box in .dae extension into robert_gazebo/meshes/ directory

Upload the collision models of fire-resque box in .dae extension into robert_gazebo/meshes/ directory

Upload models of fire-resque box in .sdf extension into robert_gazebo/models/fire_resque_box/ directory

Upload models of fire-resque box in .config extension into robert_gazebo/models/fire_resque_box/ directory

### PR DESCRIPTION
**Pull request by v-lavrushko**
Fire rescue box done
closes #12

### CODE REVIEW

File: robert_gazebo/models/fire_rescue/model.sdf [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+            <collision name="collision">
+                <geometry>
+                    <mesh>
+                        <uri>model://robert_gazebo/meshes/collision_fire_rescue_box.dae</uri>
+                        <scale>1 1 1</scale>
+                    </mesh>
+                </geometry>
+            </collision>
+        </link>
```
Comment (AndriyPt) [inline robert_gazebo/models/fire_rescue/model.sdf]: please add inertia properties

File: robert_gazebo/worlds/ucu.world [hunk 2]
```diff
@@ -10,6 +10,12 @@
       <uri>model://sun</uri>
     </include>
 
+    <include>
```
Comment (AndriyPt) [inline robert_gazebo/worlds/ucu.world]: please remove debug model

**Review by AndriyPt — CHANGES_REQUESTED** [review]

**Comment by AndriyPt** [comment]
Incorrect scale and collision model
image

File: robert_gazebo/models/fire_rescue/model.sdf [hunk 3]
```diff
@@ -0,0 +1,38 @@
+<?xml version='1.0'?>
+<sdf version='1.5'>
+    <model name="fire_rescue_box">
+        <pose>0 0 0 0 0 0</pose>
+        <static>true</static>
```
Comment (AndriyPt) [inline robert_gazebo/models/fire_rescue/model.sdf]: in order to have interaction with the object in simulation please set this property to false

**Review by AndriyPt — CHANGES_REQUESTED** [review]
Single comment and will merge

File: robert_gazebo/models/fire_rescue/model.sdf (hunk 3 above)
Comment (LAnastasiia) [reply robert_gazebo/models/fire_rescue/model.sdf]: Done

**Review by AndriyPt — APPROVED** [review]

**Pull request merged**